lib/src/user_agent_application.dart

```diff
@@ -104,77 +98,64 @@ class UserAgentApplication {
   Future<AuthResponse> ssoSilent(AuthRequest request) async {
     final response = await _convertMsalPromise<interop.AuthResponse>(
         _callJsMethod(() => _jsObject.ssoSilent(request._jsObject)));
-
     return AuthResponse._fromJsObject(response);
   }
 
   /// Gets all currently cached unique accounts based on `homeAccountIdentifier`.
-  List<Account> getAllAccounts() {
-    return _callJsMethod(() => _jsObject
-        .getAllAccounts()
+  List<Account> getAllAccounts() => _callJsMethod(() => (_jsObject.getAllAccounts() ?? [])
         .cast<interop.Account>()
         .map((jsAccount) => Account._fromJsObject(jsAccount))
         .toList());
-  }
 
   /// Gets the signed in account or `null` if no-one is signed in.
-  Account getAccount() {
-    return _callJsMethod(() {
+  Account? getAccount() => _callJsMethod(() {
       final jsAccount = _jsObject.getAccount();
-
       return jsAccount == null ? null : Account._fromJsObject(jsAccount);
     });
-  }
```

Comment:
Please keep the bracket syntax for these and some others in the file.

Reply:
I'll remove `=>` where the delegation does not fit a single line. please confirm if you wanna remove all the `=>` instances

Reply:
Yes please, I'm very against `=>` syntax for methods that aren't very straight forward and single-line.

Reply:
I believe all cases except of single-liners are fixed now, but I didn't quite understand if you wanna convert single-liners as well?

Reply:
Single-liners are fine.